tools/link.py

```diff
@@ -626,6 +626,8 @@ def phase_linker_setup(options, state, newargs):
   system_libpath = '-L' + str(cache.get_lib_dir(absolute=True))
   state.add_link_flag(sys.maxsize, system_libpath)
 
+  shared.perform_sanity_checks()
```

Comment:
IIUC this is a new check compared to before, so the link stage is now slower? That seems fine but I want to make sure I understand.

Reply:
Ah yes I guess this should be `shared.check_llvm_version`